corgi-concourse-resource/corgi_concourse_resource/corgi_api.py

```diff
@@ -1,18 +1,25 @@
 import json
-import requests
+from urllib.parse import quote
 
+import requests
 from .utils import msg
 
 
-def build_url(api_root, *args):
+def build_url(api_root, *args, **kwargs):
     parts = [api_root]
+    query = []
     parts.extend(args)
     parts = [str(p) for p in parts]
-    return "/".join(parts)
+    url = "/".join(parts)
+    for item in kwargs.items():
```

Comment:
Why not [urlencode](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode)?

Reply:
I overlooked that. Thanks, I am using it instead now.